pkg/telemetry/prometheus/packets.go

```diff
@@ -134,6 +170,27 @@ func IncrementRTCP(direction Direction, nack, pli, fir uint32) {
 	}
 }
 
+func RecordPacketLoss(direction Direction, source livekit.TrackSource, lost, total uint32) {
+	if total > 0 {
+		promPacketLoss.WithLabelValues(string(direction), source.String()).Observe(float64(lost) / float64(total) * 100)
+	}
+	if lost > 0 {
+		promPacketLossTotal.WithLabelValues(string(direction), source.String()).Add(float64(lost))
+	}
```

Comment:
I forget if we have a counter for total packets. I am guessing yes. I am trying to visualise how we will be using this.

We will get once of these readings every 10 seconds for every track on a media node.

Showing loss percentage on a node over time would be good. So, adding all total packets in a window and adding all packets lost in a window and calculating loss percentage would work (packets and packets lost are cumulative per track). But, if the window is too big (more than 10 seconds), there might be more samples from the same track. How does that affect the result? Wondering if the packets aggregation across tracks should happen in the media node and we just post the total packets and total number of losses.

I am not able to see how we will use the percentage loss from here given that it is in a 10 second window per track.

Reply:
from the distribution of track packet losses we can tell if a spike in packet loss is an isolated event or an issue with the server. from the total packet loss counter we can compute track packet loss for the server. we would need a separate total packet counter that includes the track metadata though.

i think the goal here was to get a sense of overall server health. do we want to remove this or add the track total packet counter?